stdlib/universe/quantile.go

```diff
@@ -257,8 +248,12 @@ func (a *QuantileAgg) NewUIntAgg() execute.DoUIntAgg {
 	return nil
 }
 
+// Not sure what to do here. NewFloatAgg is called in a number of places,
+// and could cause state to be recycled erroneously. Need to understand more
+// about when/how this is used.
```

Comment:
The only uses I can see are here:
https://github.com/influxdata/flux/blob/ac8d8952718cadfe5e1158cf071f8fa1bf8e38dc/execute/aggregate.go#L395
And here:
https://github.com/influxdata/flux/blob/ac8d8952718cadfe5e1158cf071f8fa1bf8e38dc/execute/aggregate.go#L259

One is used from `simpleAggregateTransformation` and the other `simpleAggregateTransformation2`. I am not sure why there are two of them. I'm not sure if this helps.